automatic/gom-player/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
```

Comment:
But I like the `;` 😛 
But honestly, it's just a habit from C#, I'm more surprised when I don't add it.
I'll remove it before merging though.